### PR TITLE
Hf vel mod try2

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

**Comment by jpa198** [comment]
Separates hf 1dvm from srf 1dvm, allowing individual perturbation of each. Just want an extra pair of eyes as an idiot check

File: srf_generation/source_parameter_generation/common.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from os.path import abspath, join, dirname
 
 import pandas as pd
+from numpy import digitize
```
Comment (jasonmotha) [inline srf_generation/source_parameter_generation/common.py]: why not import numpy like np?

Comment (jpa198) [reply srf_generation/source_parameter_generation/common.py]: It only needs digitize and asarray, I suppose it wouldn't hurt or help to change it. I can if you want

Comment (jasonmotha) [reply srf_generation/source_parameter_generation/common.py]: Yeah. I don't think it'll hurt performance. It'll just make it a little clearer that these are numpy functions.

**Pull request merged**